src/lang/index.js

```diff
@@ -1,4 +1,76 @@
-// TODO read/write language setting to local cookie
-import en from './en.json';
+const savedLang = window.localStorage && window.localStorage.getItem('localization');
+const lang = require(`./${savedLang || 'en'}.json`);
 
-export default en;
+export const langs = [
+  {
+    value: 'en',
+    translated: lang.language_en,
+    native: 'English',
```

Comment:
I see what you mean but I would rather just do it in here for simplicity. Plus, it allows us to refactor at some point to not bundle all the languages together